packages/ez-core/src/utils/map.ts

```diff
@@ -62,17 +63,33 @@ export const getGeoFeatureDataDict = (
     const datum = data.find(
       datum => datum[geoDomainKey] === properties[geoDomainKey]
     );
+
     return {
       feature,
       datum,
     };
   }, {} as GeoFeatureDataDict);
 };
 
+export let sqrtScale = d3Scale
+  .scaleSqrt()
+  .domain([0, 100])
+  .range([0, 30]);
+
+export const scaler = (rangeMin: number, rangeMax: number, value: number) => {
+  const sqrtScale = d3Scale
+    .scaleSqrt()
+    .domain([0, 100])
+    .range([rangeMin, rangeMax]);
+
+  return sqrtScale(value);
+};
+
```

Comment:
We shouldn't add scales this way. Whenever there's a new scale we should add it to the src/components/scales just like LinearScale.tsx you could add SqrtScale.tsx. It will be a context provider for children components.